.flake8

```diff
@@ -1,3 +1,8 @@
 [pycodestyle]
```

Comment:
Is the pycodestyle configuration still needed even though the flake8 style is used instead now?

Reply:
No it is not. Left it in accidentially.